products/netapp.md

```diff
@@ -0,0 +1,72 @@
+    eoas: 2027-07-31
+    eol: 2029-07-31
+    eoes: 2032-07-31
+    link: "https://docs.netapp.com/us-en/ontap/release-notes/whats-new-9151.html" # we don't have acces to full release notes
+-   releaseCycle: "9.14.1"
+    releaseDate: 2024-01-01 # estimated date from https://docs.netapp.com/us-en/ontap/release-notes/release-support-reference.html
+    eoas: 2027-01-31
+    eol: 2029-01-31
+    eoes: 2032-01-31
+    link: "https://docs.netapp.com/us-en/ontap/release-notes/whats-new-9141.html" # we don't have acces to full release notes
+-   releaseCycle: "9.13.1"
+    releaseDate: 2023-06-01 # estimated date from https://docs.netapp.com/us-en/ontap/release-notes/release-support-reference.html
+    eoas: 2026-06-30
+    eol: 2028-06-30
+    eoes: 2031-06-30
+    link: "https://docs.netapp.com/us-en/ontap/release-notes/whats-new-9131.html" # we don't have acces to full release notes
+-   releaseCycle: "9.12.1"
+    releaseDate: 2023-02-01 # estimated date from https://docs.netapp.com/us-en/ontap/release-notes/release-support-reference.html
+    eoas: 2026-02-28
+    eol: 2028-02-28
+    eoes: 2031-02-28
+    link: "https://docs.netapp.com/us-en/ontap/release-notes/whats-new-9121.html" # we don't have acces to full release notes            
+-   releaseCycle: "9.11.1"
+    releaseDate: 2022-07-01 # estimated date from https://docs.netapp.com/us-en/ontap/release-notes/release-support-reference.html
+    eoas: 2025-07-31
+    eol: 2027-07-31
+    eoes: 2030-07-31
+    link: "https://docs.netapp.com/us-en/ontap/release-notes/whats-new-9111.html" # we don't have acces to full release notes
+-   releaseCycle: "9.10.1"
+    releaseDate: 2022-01-01 # estimated date from https://docs.netapp.com/us-en/ontap/release-notes/release-support-reference.html
+    eoas: 2025-01-31
+    eol: 2027-01-31
+    eoes: 2030-01-31
+    link: "https://docs.netapp.com/us-en/ontap/release-notes/whats-new-9101.html" # we don't have acces to full release notes
+-   releaseCycle: "9.9.1"
+    releaseDate: 2021-06-01 # estimated date from https://docs.netapp.com/us-en/ontap/release-notes/release-support-reference.html
+    eoas: 2024-06-30
+    eol: 2026-06-30
+    eoes: 2029-06-30
+    link: "https://docs.netapp.com/us-en/ontap/release-notes/whats-new-991.html" # we don't have acces to full release notes          
+---
+
+> [NetApp ONTAP](https://www.netapp.com/data-storage/ontap/) is a storage operating system designed for managing and protecting data across hybrid cloud environments. It offers features like data protection, storage efficiency, and seamless scalability. 
+
+NetApp typically provides major releases of ONTAP twice per year, with regular patches and updates to address security and performance issues. Each version of ONTAP is generally supported for  8 years as follow : 
+* Full support 3 years
+* Limited Support 2 years
+* Self-Service Support 3 years
```

Comment:
A short explanation of what is Full / Limited / Self-service support would be nice for our users.